package.json

```diff
@@ -48,8 +48,8 @@
     "@hexonet/semantic-release-github-whmcs-config": "^1.0.0"
   },
   "scripts": {
-    "codeCheck": "phpcs --ignore=\"*/vendor/*\" --standard=PSR2 -q -n --colors registrars tests",
-    "codeFix": "phpcbf --ignore=\"*/vendor/*\" --standard=PSR2 -q registrars tests",
+    "codeCheck": "phpcs --ignore=\"*/vendor/*\" --standard=PSR1,PSR2 -q -n --colors registrars tests",
```

Comment:
please cleanup --ignore parameter in both lines, not necessary because no composer.json is available.
commit + squash =) if you need help with squashing, just let me know! =)